### PR TITLE
zuzana.json

### PR DESCRIPTION
**Pull request by nazuzunana**

### CODE REVIEW

File: people/zuzana.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{
+
+  "key": "zuzana",
+
+  "name": "Zuzana Červínová",
```
Comment (tomaskourim) [inline people/zuzana.json:5]: nemame tady nahodou mit i titul?

File: people/zuzana.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  "title_cs": "JUNIOR FRONTEND DEVELOPER",
+
+  "title_en": "JUNIOR FRONTEND DEVELOPER",
+
+  "short_desc_en": "Zuzana is a Bachelor of Arts and a graduate of the Czechitas Digital Academy: Web. At Mild Blue she works as a junior frontend developer.",
+
+  "short_desc_de": "Zuzana ist Bachelor of Arts und Absolventin der Czechitas Digital Academy: Web. Bei Mild Blue arbeitet sie als Junior Frontend Developer.
+  ",
+
+  "short_desc_cs": "Zuzana je bakalář umění a absolvent kurzu Digitální akademie: Web v Czechitas. Ve společnosti Mild Blue pracuje jako junior frontend developer."
```
Comment (tomaskourim) [inline people/zuzana.json:20]: a nechces to trochu rozvest? Treba neco jako.
Zuzana vystudovala to a to, ale po case zjistila, ze ji vlastne bavi programovat a delat hezke weby. Tak se prihlasila do kurzu, naucila se spoustu veci (takovych a makovych) a ted prichazi cas pretavit teorii v praxi. 
Jakoze mi prijde fajn, kdyz jsou ty medailonky takove vic osobni

**Review by tomaskourim — CHANGES_REQUESTED** [review]

File: people/zuzana.json [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+  "email": "zuzana.cervinova@mild.blue",
+
+  "title_cs": "JUNIOR FRONTEND DEVELOPER",
+
+  "title_en": "JUNIOR FRONTEND DEVELOPER",
+
+  "short_desc_en": "Zuzana is a Bachelor of Arts and a graduate of the Czechitas Digital Academy: Web. The skills she gained during the course, as well as her artistic sense, Zuzana now uses at Mild Blue, where she works as a junior frontend developer.",
+
+  "short_desc_de": "Zuzana ist Bachelor of Arts und Absolventin der Czechitas Digital Academy: Web. Ihre im Studium erworbenen Kenntnisse sowie ihr künstlerisches Gespür werden nun bei Mild Blue als Junior Frontend Developer eingesetzt und weiterentwickelt.
+  ",
```
Comment (kubantjan) [inline people/zuzana.json:18]: nemuzes mit uvozkovky na zacatku radku v jsonu. takhle je to nevalidni (i github ti to zabarvuje cervene  )

**Review by kubantjan — CHANGES_REQUESTED** [review]

**Comment by kubantjan** [comment]
Fix jsonu, jinak ok

File: people/zuzana.json [hunk 4]
```diff
@@ -0,0 +1,21 @@
+{
```
Comment (tomaskourim) [inline people/zuzana.json:1]: jeste se musis pridat do people.json

File: people/zuzana.json [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+
+  "nick": "Zuzana",
+
+  "email": "zuzana.cervinova@mild.blue",
+
+  "title_cs": "JUNIOR FRONTEND DEVELOPER",
+
+  "title_en": "JUNIOR FRONTEND DEVELOPER",
+
+  "short_desc_en": "Zuzana is a Bachelor of Arts and a graduate of the Czechitas Digital Academy: Web. The skills she gained during the course, as well as her artistic sense, Zuzana now uses at Mild Blue, where she works as a junior frontend developer.",
```
Comment (tomaskourim) [inline people/zuzana.json:15]: Jenom AJ grammar
The skills she gained during the course, as well as her artistic sense, Zuzana now uses at Mild Blue, where she works as a junior frontend developer.

->

She now uses the skills gained during the course as well as her artistic sense at Mild Blue, where she works as a junior frontend developer.

**Review by tomaskourim — DISMISSED** [review]

**Review by kubantjan — CHANGES_REQUESTED** [review]
Jeste prosim rebase nad master (nekdo uz tam neco mergnul pred tebou) (kdybys s gitem a rebasem bojovala, dej vedet) 

Jinak za me vypada dobre. 

jeste zminim ze good practice je vzdy kdyz dotahnes a cekas na review od nekoho tak dat re-request review aby ho to pinglo (ja jsem re-request ted nemel :)

**Review by kubantjan — APPROVED** [review]

**Review by kubantjan — APPROVED** [review]

**Pull request merged**